docs/source/operations/projections/peirce_q_d.rst

```diff
@@ -0,0 +1,54 @@
+.. _peirce_q_d:
```

Comment:
if keeping a new projection method, it needs to be referenced in docs/source/operations/projections/index.rst

Reply:
done